test/format/format.h

```diff
@@ -242,8 +242,8 @@ typedef struct {
 
     char *config_open; /* Command-line configuration */
 
-    TABLE *base_mirror;  /* First mirrored table */
-    bool mirror_fix_var; /* Special case if mirroring both FIX and VAR tables */
+    TABLE *base_mirror;    /* First mirrored table */
+    bool mirror_col_store; /* Special case if mirroring both FIX and VAR tables */
```

Comment:
The comment needs to be updated to **column store** tables instead of **both**.

Reply:
done! [e17a054](https://github.com/wiredtiger/wiredtiger/pull/9523/commits/e17a054a2412208eb40c14b4e06aa081d38a130c)